torch/fx/graph_module.py

```diff
@@ -276,6 +264,113 @@ def _gen_model_repr(module_name: str, module: torch.nn.Module) -> Optional[str]:
             warnings.warn("Was not able to save the following children modules as reprs -"
                           f"saved as pickled files instead: {blobified_modules}")
 
+    def has_submodule(self, target: str) -> bool:
+        """
+        Returns whether or not this GraphModule contains the submodule
+        given by ``str``.
```

Comment:
We probably want to give examples of fully-qualified names in these docstrings. It's a prominent PyTorch concept but I'm not sure it's well known (it's not really encountered in normal PyTorch usage)

Reply:
Probably want to say "given by ``target``"